components/src/core/components/ListTable/ListTable.vue

```diff
@@ -238,6 +252,18 @@ export default defineComponent({
       })),
     );
 
+    const computedItems = computed(() => {
+      if (props.loading) {
+        const defaultValue = {};
+        props.headers.forEach(header => {
+          defaultValue[header.name] = null;
+        });
+        return Array(10).fill(defaultValue);
```

Comment:
Shouldn't we make this an option rather than hardcoding as 10 so if someone needs it, they can change it? Will it have any adverse effects?

Reply:
can do, i'll make the change

Reply:
done